src/pages/unsere-formate.module.css

```
@@ -2,14 +2,155 @@
+    margin-bottom: 3rem;
+}
+
+.categoryTitle {
+    color: var(--ifm-color-primary);
+    font-size: 1.5rem;
+    font-weight: 600;
+    margin-bottom: 1.5rem;
+    padding-left: 1rem;
+    border-left: 4px solid var(--ifm-color-primary);
+    background: var(--ifm-color-emphasis-100);
+    padding: 1rem;
```

Biome (2.1.2)

[error] Unexpected shorthand property padding after padding-left



(lint/suspicious/noShorthandPropertyOverrides)